test/statements/sungeun/bad_label.good

```diff
@@ -1,3 +1,3 @@
-bad_label.chpl:7: error: can only label for-, while-do- and do-while-statements
-bad_label.chpl:13: error: can only label for-, while-do- and do-while-statements
-bad_label.chpl:20: error: can only label for-, while-do- and do-while-statements
+bad_label.chpl:7: error: cannot label 'EmptyStmt' statement
+bad_label.chpl:13: error: cannot label 'EmptyStmt' statement
+bad_label.chpl:20: error: cannot label 'EmptyStmt' statement
```

Comment:
Hmm, is this the output we want? `EmptySmt`, is that an implementation detail?

Reply:
This test has label statements followed by semicolons, and since labels are not terminated by semicolons the semicolon constitutes an `EmptyStmt`. So yes, an implementation detail; I'll change it to emit a different error message in this case that specifies you don't terminate a label with a semicolon.